scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/executors/forked/env/ForkedTaskVariablesManager.java

```diff
@@ -133,11 +134,13 @@ public RemoteSpace createDataSpaceNodeClient(TaskContext container, SchedulerNod
     }
 
     public void replaceScriptParameters(Script<?> script, Map<String, String> thirdPartyCredentials,
-            VariablesMap variables, PrintStream errorStream) {
+            VariablesMap variables, Map<String, Serializable> jobResults, PrintStream errorStream) {
```

Comment:
no need to modify this method, we don't want to replace any script parameters using the (empty) jobResultMap